documentation/src/main/asciidoc/introduction/Introduction.adoc

```diff
@@ -13,7 +13,7 @@ Hibernate makes *relational data* visible to a program written in Java, in a *na
 3. allowing performance optimizations to be made after the basic persistence logic has already been written.
 ****
 
-Here the relational data is the focus, along with the importance of typesafety.
+Here the relational data is the focus, along with the importance of type safety.
```

Comment:
not sure the change is valid, but "type safety" was used later in the same chapter.

Reply:
yeah I go back and forth on this. I like "typesafety" as a word, but I accept that "type safety" is more standard.